### PR TITLE
Token Picker Component

### PR DESCRIPTION
**Pull request by eupston**
Initial Token Picker used across pool and swap

### CODE REVIEW

File: components/shared/TokenPicker.tsx [hunk 1]
```diff
@@ -90,66 +83,121 @@ const TokenPicker: React.FC<{
...
+								);
+							})}
+						</div>
+					)}
+				</div>
+			</FormControl>
+			<div className={styles.amountContainer}>
+				<Button
+					sx={{
+						fontFamily: "Roboto",
```
Comment (ken-centrality) [inline components/shared/TokenPicker.tsx:179]: I've seen this across different components, do we need to specify `fontFamily` explicitly for some components? @eupston @aidan-starke

**Review by ken-centrality — APPROVED** [review]
Looks good, just a question about `fontFamily`

File: components/shared/TokenPicker.tsx (hunk 1 above)
Comment (aidan-starke) [reply components/shared/TokenPicker.tsx:179]: not as far as I know, we could also remove the fonts file from public, and the references to Teko in theme/theme.ts and theme/StyledComponents.ts

**Review by aidan-starke — APPROVED** [review]
looking awesome man!

once this is merged I will make some changes on Pool - balances and setMax operate differently on withdraw side

**Pull request merged**